docs/core/stories.rst

```diff
@@ -22,12 +28,11 @@ This is a guide to the story data format.
 Format
 ------
 
-Here's an example from the `bAbI <https://research.fb.com/downloads/babi/>`_
-dataset (converted into Rasa stories):
+Here's an example of a dialogue in the Rasa story format:
 
 .. code-block:: story
 
-   ## story_07715946    <!-- name of the story - just for debugging -->
+   ## greet + location/price + cuisine + num people    <!-- name of the story - just for debugging -->
```

Comment:
```suggestion
   ## cheap_spanish_cuisine_in_rome    <!-- name of the story - just for debugging -->
```

Reply:
I find the `+` naming style more convenient to quickly get a sense of the story order and structure 😄.

Reply:
I see your point.  Hopefully we can make this more human-readable though

What about `greet, then location & price, then cuisine, then num people` ?